core.go

```diff
@@ -119,8 +125,12 @@ type (
 	// If on-demand device provisioning is enabled for your MODE project, you
 	// can call ProvisionDevice to create a new DeviceContext.
```

Comment:
Please update the GoDoc on the new TLS-related fields in DeviceContext.

Reply:
updated.